codalab/worker/local_run/local_dependency_manager.py

```diff
@@ -186,7 +186,7 @@ def _cleanup(self):
             with self._global_lock:
                 self._acquire_all_locks()
                 bytes_used = sum(dep.size_bytes for dep in self._dependencies.values())
-                serialized_length = len(codalabworker.pyjson.dumps(self._dependencies))
+                serialized_length = len(codalab.pyjson.dumps(self._dependencies))
```

Comment:
Back to my comment on the original PR: why do we have codalab.pyjson instead of codalabworker.pyjson?

Reply:
This is different than the original PR. I merged something wrong in this PR. It should be `codalab.worker`